src/main/java/com/easypost/model/Children.java

```diff
@@ -0,0 +1,12 @@
+package com.easypost.model;
+
+import java.util.List;
+
+public class Children extends EasyPostResource {
+    private String parentId;
+    private String name;
+    private String phoneNumber;
+    private Boolean verified;
+    private Boolean defaultCarbonOffset;
```

Comment:
question: is this still valid? I thought this key was gone

Reply:
Good catch, I will remove this